requirements.in

```diff
@@ -2,3 +2,4 @@ certifi >= 14.05.14
 six >= 1.10
 python_dateutil >= 2.5.3
 urllib3 >= 1.15.1
+kagglesdk

```

Comment:
Steve - do you know whether `requirements.in` and `requirements.txt` are still required?  During local testing, it worked fine with just the `pyproject.toml` changes.  Can we remove these files?